project/src/main/puzzle/wobbler.gd

```diff
@@ -48,22 +48,24 @@ func _physics_process(delta: float) -> void:
 
 func set_base_scale(new_base_scale: Vector2) -> void:
 	base_scale = new_base_scale
-	if is_inside_tree():
-		_refresh_scale()
+	_refresh_scale()
 
 
 func set_base_rotation(new_base_rotation: float) -> void:
 	base_rotation = new_base_rotation
-	if is_inside_tree():
-		_refresh_rotation()
+	_refresh_rotation()
 
 
 func _refresh_scale() -> void:
+	if not is_inside_tree():
+		return
```

Comment:
Document the purpose of this null protection (math reasons, divide by null/divide by zero)

Reply:
Added comment, `# avoid errors dividing by zero/null before fields are initialized`